models/load_redeemer.py

```diff
@@ -9,6 +9,17 @@
 from models.load_utils import to_markdown
 
 
+def remove_strings(page_content: str) -> str:
```

Comment:
See my comment in the internship channel. Rather than asking everyone to modify each loader to remove markdown, I will remove it when generating embeddings during indexing.  I'm sorry to make you go to unnecessary work. I don't think this PR is needed.